components/Video/MutedVideoLoop.tsx

```diff
@@ -0,0 +1,10 @@
+import { forwardRef } from 'react';
+import { Video, type VideoProps } from './Video';
+
+/**
+ * This component is a more specific version of the Video component created for convenience.
+ * It is used for decorative muted video loops.
+ */
+export const MutedVideoLoop = forwardRef<HTMLVideoElement, VideoProps>((props, ref) => (
+  <Video ref={ref} role="presentation" muted autoPlay loop playsInline {...props} />
+));
```

Comment:
A convenience component for muted loops so I don't have to write out those same attributes all the time.